docker-compose.yml

```diff
@@ -18,6 +18,22 @@ services:
       - "18250:18250"
     volumes:
       - ./:/blockstore/app/:cached
+    depends_on:
+      - "datastore"
+
+  datastore:
+    container_name: datastore
+    image: mysql:8.0
+    ports:
+        - "3306:3306"
+    volumes:
+      - ./datastore_volume:/var/lib/mysql
```

Comment:
Do we plan on making the volume map to a local file? I think the syntax you're looking for is:

```yml
    ...
    volumes:
      - datastore_volume:/var/lib/mysql
    ...

networks:
  devstack:
    external:
      name: devstack_default
volumes:
  datastore_volume:
```